src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/IPCServicesTest.java

```diff
@@ -570,7 +572,7 @@ void GIVEN_LifeCycleEventStreamClient_WHEN_update_state_THEN_service_state_chang
     @SuppressWarnings({"PMD.CloseResource", "PMD.AvoidCatchingGenericException"})
     @Test
     void GIVEN_LifeCycleEventStreamClient_WHEN_subscribe_to_component_update_THEN_service_receives_update_and_close_stream() throws Exception {
-
+        LogConfig.getRootLogConfig().setLevel(Level.DEBUG);
```

Comment:
nit: is this necessary?

Reply:
@MikeDombo says it is - so let's add a comment about why so it's obvious what this is doing for the test that other's don't need